templates/security/certifications.html

```diff
@@ -10,10 +10,10 @@
 <section class="p-strip--suru-topped">
  <div class="row u-equal-height u-vertically-center">
    <div class="col-8">
-     <h1>Security Certifications</h1>
-     <h2 class="p-heading--3">Deploy regulated and high security workloads</h2>
-     <p>Regulated and high security environments have complex requirements as they are designed to tackle many threats, known and unknown. Whatever path you have chosen in navigating compliance with a cybersecurity framework, such as the ones defined by ISO 27000, NIST, PCI or CIS Controls, Canonical provides you with a secure and easy to use operating system, that enables your compliance and reduces operational risk, whether on-premise or on a public cloud. We, at Canonical, are transparent on the security features we enable, provide access and automation to Industry accepted hardening and compliance profiles, such as CIS and DISA-STIG as well as present 3rd party attestation and certifications like FIPS 140-2 and Common Criteria.</p>
-     <p><a class="p-button--positive js-invoke-modal" href="/support/contact-us">Contact us</a></p>
+     <h1>Security Certifications & Hardening</h1>
+     <h2 class="p-heading--3">Run regulated and high security workloads on Ubuntu</h2>
+     <p>Whatever cybersecurity framework you have chosen, including ISO 27000, NIST, PCI or CIS Controls, <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> enable your compliance and reduce your operational risk. Access automation for hardening and compliance profiles, such as CIS and DISA-STIG as well as the FIPS 140-2 and Common Criteria certifications.</p>
+     <p><a class="p-button--positive js-invoke-modal" href="/security/contact-us">Contact us</a></p>
```

Comment:
Contact form is wrong, in the copy docs is `/security/certifications#get-in-touch`

Reply:
Don't link to `#get-in-touch` this is purely a JS interaction that is handled via the `js-invoke-model` class. The link should link to a backup static form for non-JS.

Reply:
I spoke to the pm about it but I think there was a misunderstanding, so in the end just linked to security/contact-us as it seemed most appropriate.